dialect_redshift.go

```diff
@@ -0,0 +1,19 @@
+package gorp
+
+type RedshiftDialect struct {
+	PostgresDialect
+}
+
+func (d RedshiftDialect) InsertAutoIncrToTarget(exec SqlExecutor, insertSql string, target interface{}, params ...interface{}) error {
```

Comment:
nit, but mind changing `target` to `_` to communicate that that's not anything we care about?

Reply:
9604ea7